src/__init__.py

```diff
@@ -282,6 +283,9 @@ def create_item(self, name: str, class_override: Optional['ItemClassification']=
         else:
             classification = ItemClassification.filler
 
+        if "classification" in item and item["classification"]:
+            classification = reduce((lambda a, b: a | b), {ItemClassification[str_classification.strip()] for str_classification in item["classification"].split(",")})
+
```

Comment:
missing and else: here so the other base classification is never run currently
```suggestion
        if "classification" in item and item["classification"]:
            classification = reduce((lambda a, b: a | b), {ItemClassification[str_classification.strip()] for str_classification in item["classification"].split(",")})
        else:
```